requirements.yml

```diff
@@ -2,3 +2,4 @@
 collections:
   - name: middleware_automation.common
   - name: ansible.posix
+  - name: community.general # for `maven_artifact`
```

Comment:
ouch; that's a big no-no for the downstream version of the collection

Reply:
hmm, any other options that come to your mind? At the end of the day it's just a plain http download w/ authentication, but re-inventing the wheel wrt. authn against github repo seems more than smelly...

Reply:
I am considering forking the module inside middleware_automation.common (redhat.runtimes_common), so it can go supported. It would also lower the changeset bar down from breaking_change to minor_change; please continue your work, I'll find a solution

Reply:
Please also note the addition of `lxml` in `requirements.txt`, so I guess this is breaking for users already making use of providers on way or the other...

Reply:
We already have lxml in the common dependency reqs :); but i'll remove the boto dependency for s3 in the forked module. I'd say it marks it has a major change (user may have to make changes if they were already installing providers by themselves)

Reply:
You can already do the change in this PR please:
community.general.maven_artifact -> middleware_automation.common.maven_artifact

Reply:
https://github.com/ansible-middleware/common/pull/24

Reply:
All done.